api/v1/swagger.yaml

```diff
@@ -7537,6 +7537,61 @@ paths:
           $ref: "#/responses/genericError"
 
 # =============================================================================
+  /botData:
+    x-swagger-router-controller: botData
+    post:
+      security:
+        - jwt: []
+      summary: Create a new botData
+      tags: [ botData ]
+      description: Create new bot data for a rooms specific bots
+      operationId: postBotData
+      parameters:
+        -
+          name: queryBody
+          in: body
+          required: true
+          schema:
+            type: object
+            description: Create botData properties
+            properties:
+              name:
+                type: string
```

Comment:
Putting them in swagger means fail faster--swagger middleware sends back error instead of having the request wind its way through the middleware to our controller and through to the db.